tests/test_config.py

```diff
@@ -1,17 +1,61 @@
+import os
 import ssl
 
 import pytest
 
 import httpcore
 
 
+ASSETS = os.path.join(os.path.dirname(__file__), "assets")
+
+
 @pytest.mark.asyncio
 async def test_load_ssl_config():
     ssl_config = httpcore.SSLConfig()
     context = await ssl_config.load_ssl_context()
     assert context.verify_mode == ssl.VerifyMode.CERT_REQUIRED
 
 
+@pytest.mark.asyncio
+async def test_load_ssl_config_verify_non_existing_path():
+    ssl_config = httpcore.SSLConfig(verify="/path/to/nowhere")
+    with pytest.raises(IOError):
+        await ssl_config.load_ssl_context()
```

Comment:
These seem legit, but let's try to keep different slices of stuff (eg. test cases for SSL) as different pull requests.